mdfreader/mdf3reader.py

```diff
@@ -1256,7 +1262,19 @@ def writePointer(f, pointer, value):
                 # channel description
                 desc = self.getChannelDesc(channel)
                 writeChar(fid, desc, size=127)  # channel description
-                fid.write(pack(UINT16, bitOffset))  # bit position
+
+                #if bitOffset exceeds two byte limit, we start using the byte offset field
+                if (bitOffset > 0xFFFF):
+                    bitOffset -= 0x10000
+                    byteOffset += 8192
+
+                #print("working on {} at {} byteoffset {}".format(channel, bitOffset, byteOffset))
+
+                try:
+                    fid.write(pack(UINT16, bitOffset))  # bit position
```

Comment:
This try can probably be removed now.  The reason this was added it because of an overflow of bitOffset when it exceeded the limits of UINT16.  This should not happen anymore.